v3/src/data-interactive/handlers/component-handler-slider.test.ts

```diff
@@ -0,0 +1,61 @@
+import { appState } from "../../models/app-state"
+import { toV3Id } from "../../utilities/codap-utils"
+import { DIComponentInfo } from "../data-interactive-types"
+import { diComponentHandler } from "./component-handler"
+import { getGlobalValueManager, getSharedModelManager } from "../../models/tiles/tile-environment"
+import { kSliderIdPrefix } from "../../components/slider/slider-registration"
+import { ISliderModel, isSliderModel } from "../../components/slider/slider-model"
+import { AnimationDirections, AnimationModes } from "../../components/slider/slider-types"
+
+
+describe("DataInteractive ComponentHandler Slider", () => {
+  const documentContent = appState.document.content!
+  const globalManager = getGlobalValueManager(getSharedModelManager(documentContent))
+  globalManager?.addValueSnapshot({ name: "global", value: 10 })
+  const handler = diComponentHandler
+
+  it("create slider works", async () => {
+    // Create slider with no value specified
+    expect(documentContent.tileMap.size).toBe(0)
+    const newValueResult = handler.create!({}, { type: "slider" })
+    expect(newValueResult.success).toBe(true)
+    expect(documentContent.tileMap.size).toBe(1)
+    const newValueResultValues = newValueResult.values as DIComponentInfo
+    const newValueTile = documentContent.tileMap.get(toV3Id(kSliderIdPrefix, newValueResultValues.id!))!
+    expect(newValueTile).toBeDefined()
+    expect(isSliderModel(newValueTile.content)).toBe(true)
+
+    // Delete slider tile
+    const deleteResult = handler.delete!({ component: newValueTile })
+    expect(deleteResult.success).toBe(true)
+    expect(documentContent.tileMap.size).toBe(0)
+
+    // Create slider with value and other options specified
+    const oldValueResult = handler.create!({}, {
+      type: "slider", globalValueName: "global", lowerBound: -100, upperBound: 100,
+      animationDirection: 0, animationMode: 1
+    })
```

Comment:
Yes, v2 allows you to specify the global value, which I believe is the only way to create a slider for a global created via the `create global` API call. It's definitely a bit of a funky system, especially since closing the slider deletes the global in v2, but that's how it works, so that's what I replicated.